charts/brig/values.yaml

```diff
@@ -99,6 +99,10 @@ config:
     providerTokenTimeout: 900
     legalholdUserTokenTimeout: 4838400
     legalholdAccessTokenTimeout: 900
+  sft:
```

Comment:
This block doesn't have any conditions: even if secrets.sftTokenSecret is not set it would configure with a secret pointing to an non-existing file. This block should probably have the same condition as the secret file.